dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/specific/uploadconfig/elements/UploadConfigElement.kt

```diff
@@ -0,0 +1,25 @@
+package org.dataland.frameworktoolbox.specific.uploadconfig.elements
+
+/**
+ * A single element of a MLDT View-Configuration (either Cell or Section)
+ */
+sealed interface UploadConfigElement {
+    val parentSection: SectionUploadConfigBuilder?
+
+    /**
+     * Returns true iff this is an instance of a section (required for FreeMarker)
+     */
+    fun isSection() = this is SectionUploadConfigBuilder
+
+    /**
+     * Returns true iff this is an instance of a cell (required for FreeMarker)
```

Comment:
Here are typos in the docs. Should be if instead of iff

Reply:
I also thought so at first. But:
`"Iff" stands for "if and only if" and is a logical operator used in programming and mathematics.`
@MarcTM01  is this a typo or intentional?

Reply:
Intentional. It seems that the writing I'm doing for my theoretical computer-science classes has found it's way into how I write documentations :D

Reply:
Ok! Thank you, again i've learned something new!